core/jobs_test.py

```diff
@@ -779,20 +779,21 @@ def test_validity_of_each_continuous_computation_class(self):
                         event_type),
                     event_services.BaseEventHandler))
 
-            rdc = klass._get_realtime_datastore_class()  # pylint: disable=protected-access
-            self.assertTrue(issubclass(
-                rdc, jobs.BaseRealtimeDatastoreClassForContinuousComputations))
-
             # The list of allowed base classes. This can be extended as the
             # need arises, though we may also want to implement
             # _get_continuous_computation_class() and
             # entity_created_before_job_queued() for other base classes
             # that are added to this list.
             allowed_base_batch_job_classes = [
                 jobs.BaseMapReduceJobManagerForContinuousComputations]
+            rdc = klass.get_event_types_listened_to(
+                test_only_get_realtime_datastore=True)
             self.assertTrue(any([
-                issubclass(klass._get_batch_job_manager_class(), superclass)  # pylint: disable=protected-access
+                issubclass(klass.get_event_types_listened_to(
+                    test_only_get_batch_job_manager=True), superclass)  # pylint: disable=protected-access
```

Comment:
You should be able to get rid of the pylint disable statement now.

Reply:
@kevinlee12 , I've made the change